test/config/examples/cifar10-pytorch-adl.yml

```diff
@@ -0,0 +1,23 @@
+authorName: nni
+experimentName: default_test
+maxExecDuration: 15m
+maxTrialNum: 1
+trialConcurrency: 1
+searchSpacePath: ./cifar10_adl_search_space.json
+
+tuner:
+  builtinTunerName: Random
+assessor:
+  builtinAssessorName: Medianstop
+  classArgs:
+    optimize_mode: maximize
+trial:
+  codeDir: ../../../examples/trials/cifar10_pytorch
```

Comment:
Can we just set it as `/examples/trials/cifar10_pytorch`?

Reply:
updated.

Reply:
Thanks. The rest LGTM.